### PR TITLE
33472: Logic Strings Const-afied | Login Component Split Up

### LINKED ISSUE #33472: Reduce amount of "magic strings" on FE
## Issue Description
There are numerous areas in the Identity code space where non `const` strings are used as arguments for logic. These should be updated as much as possible to use values stored in a `constants` file.

This will reduce the likelihood of typo based bugs

---
## Tasks
- [ ] Find areas where "magic strings" are used most frequently
- [ ] Update these occurrences with constants

## Acceptance Criteria
- [ ] Constantification of as many logic based strings as possible
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

### PR DESCRIPTION
**Pull request by Kwoly**
## Description
May auth related strings are used in logic, and retyped over and over again. This PR aims to address this by utilizing constants and removing the redundancy and chance for typos.

While doing this, I also tackled some DRY issues with the Login components

## Original issue(s)
Closes department-of-veterans-affairs/va.gov-team#33472

## Testing done
- Unit tests are still passing 

## Acceptance criteria
- [x] Auth related strings moved to constants
- [x] Login Components split up and made more dry

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/utilities/sso/keepAliveSSO.js [hunk 1]
```diff
@@ -1,36 +1,20 @@
...
-      LEVEL: Sentry.Severity.Info,
-    },
-    default: {
-      LEVEL: Sentry.Severity.Error,
-    },
-  };
 
-  const LEVEL = caughtExceptions[message]
-    ? caughtExceptions[message].TYPE
-    : caughtExceptions.default.TYPE;
```
Comment (Kwoly) [inline src/platform/utilities/sso/keepAliveSSO.js:33]: Looks like `.TYPE` never existed and this may have not been working for some time

**Review by timwright12 — APPROVED** [review]

**Review by Mottie — APPROVED** [review]

**Review by asg5704 — APPROVED** [review]
Great job Kyle 🚀

**Review by bkjohnson — APPROVED** [review]

**Review by ddzz — APPROVED** [review]

**Comment by Kwoly** [comment]
Could I get review from the following teams please and thanks you:
- @department-of-veterans-affairs/vsa-public-websites-frontend 
- @department-of-veterans-affairs/vsa-ebenefits-frontend 
- @department-of-veterans-affairs/vsa-authd-exp-frontend 
- @department-of-veterans-affairs/vsa-debt-frontend 
- @department-of-veterans-affairs/vsa-caregiver-frontend 
- @department-of-veterans-affairs/vsa-healthcare-experience

**Review by tmitchellgcio — APPROVED** [review]

**Review by jerekshoe — APPROVED** [review]

**Review by mdewey — APPROVED** [review]
LGTM!

**Review by kelsonic — APPROVED** [review]
This doesn't touch any PW files but lgtm! 🎉 (I think we got tagged because of that modification to a static-pages/* authenticated app)

**Review by craigwheeler — APPROVED** [review]

**Review by adamwhitlock1 — APPROVED** [review]
For this test

`src/platform/user/tests/authentication/components/loginButtons.unit.spec.js`

Since loginButtons is no longer a component, should we just rename this to `loginContainer.unit.spec.js` to match the component being tested? 

Otherwise LGTM

**Pull request merged**